### PR TITLE
[hydra-cli] subscriptions for processor state updates + depenedcies management

### PR DESCRIPTION
**Pull request by dzhelezov**
After #223 

This PR adds new templates to hydra-cli so that the generated query node server supports subscriptions to the processor state update

It also changes the way dependencies are added to the generated project: instead of being hardcoded, they are taken from the "queryNodeDependencies" section of `hydra-cli/package.json` and are easier to manage (in particular, the versions).

### CODE REVIEW

File: packages/hydra-cli/package.json [hunk 1]
```diff
@@ -68,6 +68,12 @@
     "tslib": "1.11.2",
     "warthog": "https://github.com/metmirr/warthog/releases/download/v2.23.0/warthog-v2.23.0.tgz"
   },
+  "queryNodeDependencies": {
+    "lodash": "^4.17.20",
+    "pg-listen": "^1.7.0",
+    "warthog": "https://github.com/metmirr/warthog/releases/download/v2.22.0/warthog-v2.22.0.tgz",
```
Comment (metmirr) [inline packages/hydra-cli/package.json:74]: It is not the latest version.

File: packages/hydra-cli/src/templates/graphql-server/warthog.env.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 default_env: &default_env
     WARTHOG_INTROSPECTION: true
+    WARTHOG_SUBSCRIPTIONS: true
```
Comment (metmirr) [inline packages/hydra-cli/src/templates/graphql-server/warthog.env.yml:3]: do have to set this in the `.env` as well?

Comment (dzhelezov) [reply packages/hydra-cli/src/templates/graphql-server/warthog.env.yml:3]: It's not needed. `.env` is generated from the `yml` file with `warthog:configure`

Comment (metmirr) [reply packages/hydra-cli/src/templates/graphql-server/warthog.env.yml:3]: thanks!

**Review by metmirr — APPROVED** [review]

**Pull request merged**